src/store.ts

```diff
@@ -110,9 +110,9 @@ type StoresValues<T> = T extends Readable<infer U> ? U :
 /**
  * Derived value store by synchronizing one or more readable stores and
  * applying an aggregation function over its input values.
- * @param stores input stores
- * @param fn function callback that aggregates the values
- * @param initial_value when used asynchronously
+ * @param {Stores} stores input stores
+ * @param {function(StoresValues,[Subscriber])|Unsubscriber|void}fn function callback that aggregates the values
```

Comment:
Is `Subscriber` the right nomenclature here? The second argument is a `set` function

Reply:
You're right. I'll add a commit to have the 2nd argument of fn be a function.